### PR TITLE
Update language-data

### PR DESCRIPTION
**Pull request by amire80**
* Update autonyms for Minangkabau and Manipuri.
* Add Awadhi (awa)

Updating to
https://github.com/wikimedia/language-data/commit/ceec188319e8ae340724f84a3bdff6e712b38531

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -47,5 +47,8 @@
   "license": "(MIT OR GPL-2.0+)",
   "scripts": {
     "test": "grunt test --verbose"
+  },
+  "dependencies": {
+    "browserify": "^16.4.0"
```
Comment (santhoshtr) [inline package.json]: browserify is not a dependency. It is only used for transforming the fetched language-data to jquery.uls.data

The script to update language-data has a check if browserify already installed. If not it installs it. See https://github.com/wikimedia/jquery.uls/blob/master/scripts/fetch-language-data.sh#L26

Comment (amire80) [reply package.json]: I didn't add these lines. The script added them:

```
macdavid:jquery.uls aaharoni$ git fetch --all
Fetching origin
Fetching upstream
macdavid:jquery.uls aaharoni$ git checkout master
Already on 'master'
Your branch is up to date with 'origin/master'.
macdavid:jquery.uls aaharoni$ git reset --hard upstream/master
HEAD is now at 0bd037d Merge pull request #345 from wikimedia/dependabot/npm_and_yarn/eslint-utils-1.4.2
macdavid:jquery.uls aaharoni$ rm -rf node_modules/
macdavid:jquery.uls aaharoni$ rm -rf vendor/
macdavid:jquery.uls aaharoni$ git status
On branch master
Your branch is up to date with 'origin/master'.

nothing to commit, working tree clean
macdavid:jquery.uls aaharoni$ git branch -c clean-update-language-data
macdavid:jquery.uls aaharoni$ npm install

> puppeteer@1.18.1 install /Users/aaharoni/dev/jquery.uls/node_modules/puppeteer
> node install.js

Downloading Chromium r672088 - 108 Mb [====================] 100% 0.0s 
Chromium downloaded to /Users/aaharoni/dev/jquery.uls/node_modules/puppeteer/.local-chromium/mac-672088
added 591 packages from 328 contributors and audited 3592 packages in 32.035s
found 0 vulnerabilities

macdavid:jquery.uls aaharoni$ git diff
macdavid:jquery.uls aaharoni$ ./scripts/fetch-language-data.sh 
readlink: illegal option -- f
usage: readlink [-n] [file ...]
Getting latest language-data from https://github.com/wikimedia/language-data.git
Cloning into './vendor/language-data'...
remote: Enumerating objects: 62, done.
remote: Counting objects: 100% (62/62), done.
remote: Compressing objects: 100% (40/40), done.
remote: Total 380 (delta 33), reused 39 (delta 22), pack-reused 318
Receiving objects: 100% (380/380), 147.70 KiB | 796.00 KiB/s, done.
Resolving deltas: 100% (198/198), done.
Installing browserify
+ browserify@16.5.0
added 106 packages from 103 contributors in 38.595s
Transforming language-data
language-data wrote to ./src//jquery.uls.data.js
macdavid:jquery.uls aaharoni$ git status
On branch master
Your branch is up to date with 'origin/master'.

Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git checkout -- <file>..." to discard changes in working directory)

	modified:   package-lock.json
	modified:   package.json
	modified:   src/jquery.uls.data.js

no changes added to commit (use "git add" and/or "git commit -a")
macdavid:jquery.uls aaharoni$ git diff package.json 
diff --git a/package.json b/package.json
index e1d9034..b2d0693 100644
--- a/package.json
+++ b/package.json
@@ -47,5 +47,8 @@
   "license": "(MIT OR GPL-2.0+)",
   "scripts": {
     "test": "grunt test --verbose"
+  },
+  "dependencies": {
+    "browserify": "^16.5.0"
   }
 }
macdavid:jquery.uls aaharoni$ 
```

**Comment by santhoshtr** [comment]
This is because npm altered their default behaviour with `npm install`(See https://github.com/npm/npm/issues/17010)

I just submitted a PR to fix this https://github.com/wikimedia/jquery.uls/pull/346
Once that is merged, you no longer will have package.json changes.

**Comment by amire80** [comment]
I'll redo it, now that the update script is fixed.

**Pull request closed without merging**